.ci/test-cluster.yml

```diff
@@ -78,7 +78,7 @@ services:
     container_name: kuzzle_redis
 
   elasticsearch:
-    image: kuzzleio/elasticsearch:7.16.2
+    image: kuzzleio/elasticsearch:7
```

Comment:
Why change the version of es here ?

Reply:
Now, we use this version, because it is built for amd64/arm64